src/nyc/c4q/group5/TurkishPizza.java

```diff
@@ -0,0 +1,11 @@
+package nyc.c4q.group5;
+
+/**
+ * Created by meltemyildirim on 9/13/16.
+ */
+public class TurkishPizza extends PizzaSlice{
+
+  public TurkishPizza(EnumType eType, EnumCrust eCrust) {
```

Comment:
All food items must have an empty constructor. This constructor has 2 parameters